cmd/dispatch-daemon/main.go

```diff
@@ -38,11 +46,11 @@ func main() {
 		Execute(&buffer, data); err != nil {
 		config.Log().Fatal("failed to execute template: ", err.Error())
 	}
-	fmt.Println(mailing.NewMailman(mailing.SMTPParams{
-		Host:     "smtp.gmail.com",
-		Port:     587,
+	logger.Info(mailing.NewMailman(mailing.SMTPParams{
+		Host:     os.Getenv("MAILMAL_HOST"),
+		Port:     port,
 		Username: from,
-		Password: "guze dokh umzh ulvs"}).
+		Password: os.Getenv("MAILMAL_PASSWORD")}).
 		Send(mailing.Email{
 			From:     from,
 			To:       []string{"daha@gmail.com"},
```

Comment:
What does this service do? As I see, it sends letter for hardcoded mail. Give me please more context.

Reply:
Ah ok, I see that it doesn't implemented yet

Reply:
In perspective it will:
- fetch data (id and count of subscribers) of dispatches filtering by time of sending
- invoke sending of emails through gRPC

Reply:
Yes, you have been told about this in telegram, sry for my memory =)

Great, you will have a chance in the future to implement this 👍